src/components/data-table-v2/_data-table-v2-inline-edit-cell.scss

```diff
@@ -24,7 +24,345 @@
+  // the input doesn't overlap with the status
+  $spacing--cell--status: 2rem;
+
+  //----------------------------------------------------------------------------
+  // Table
+  //----------------------------------------------------------------------------
+
+  // Reset hover styles while editing
+  .#{$prefix}--data-table--editing tr:hover td {
+    background-color: inherit;
+    border-top: 1px solid $ui-04;
+    border-bottom: 1px solid $ui-04;
+  }
+
+  .#{$prefix}--data-table--editing tr:hover td:first-of-type {
+    border-left: 1px solid $ui-04;
+  }
+
+  .#{$prefix}--data-table--editing tr:hover td:last-of-type {
+    border-right: 1px solid $ui-04;
+  }
+
+  //----------------------------------------------------------------------------
+  // Cell Edit
+  //----------------------------------------------------------------------------
+
+  // Buffer is created on the right-hand side so that status indicators have the
+  // appropriate space to display.
+  .#{$prefix}--data-table-cell--editable {
+    position: relative;
+    padding-right: $spacing--cell--activity;
+  }
+
+  td.#{$prefix}--data-table-cell--editable:last-of-type {
+    // Reset the styles inherited from data-table-v2 for padding-right styles
+    // but adding our manual offset plus the spacing used before columns to get
+    // a similar look for when an editable cell is in the middle of a table.
+    padding-right: #{$spacing--cell--activity + $spacing--columns--before};
+  }
+
+  // Minor offset for the last cell in a row so that the action bar does not
+  // touch the edge of the cell
+  td.#{$prefix}--data-table-cell--editable:last-of-type > .#{$prefix}--data-table-cell__activity {
+    margin-right: $spacing--columns--before;
+  }
+
+  //----------------------------------------------------------------------------
+  // Cell Edit - Content
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-cell__content {
+    position: relative;
+    z-index: 1000;
+    // Set line-height to match that within inputs
+    line-height: normal;
+  }
+
+  //----------------------------------------------------------------------------
+  // Cell Edit - Activity
+  //----------------------------------------------------------------------------
+
+  // Styles used for getting 100% width and height on an descendant of a table
+  // cell. Display is used for alignment of most items to the end of the cell
+  .#{$prefix}--data-table-cell__activity {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    position: absolute;
+    top: 0;
+    right: 1rem;
+    bottom: 0;
+    left: 0;
+  }
+
+  //----------------------------------------------------------------------------
+  // Cell Edit - Trigger
+  //----------------------------------------------------------------------------
+
+  .#{$prefix}--data-table-cell__edit {
+    @include button-reset(false);
```

Comment:
Above you added quotes around `false`. Do we need that here as well (Unfamiliar with this parameter)

Reply:
Tbh, I'm not too sure about this parameter either. I think I copied and pasted this particular one from the stable mixin, but for the other one (if I remember correctly) it was breaking unless I added the single quotes. Gonna look into this!